src/Network/Network/ApplicationGateway/GetAzureApplicationGatewayAvailableServerVariableAndHeader.cs

```diff
@@ -0,0 +1,76 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using AutoMapper;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.WindowsAzure.Commands.Common;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Reflection;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ApplicationGatewayAvailableServerVariableAndHeader"), OutputType(typeof(PSApplicationGatewayAvailableServerVariableAndRequestHeaderResult))]
```

Comment:
`OutputType` should be `Dictionary`, judging by cmdlet's output:

```powershell
PS C:\> $result = Get-AzApplicationGatewayAvailableServerVariableAndHeader
PS C:\> $result.GetType()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Dictionary`2                             System.Object


PS C:\> $result

Key                     Value
---                     -----
AvailableServerVariable {request_query, client_ip, client_port, client_user...}
AvailableRequestHeader  {A-IM, Accept, Accept-Charset, Accept-Encoding...}
AvailableResponseHeader {Access-Control-Allow-Origin, Access-Control-Allow-Credentials, Access-Control-Expose-Headers, Access-Control-Max-Age...}
```

You can also add return type check to tests, like it's done here:
https://github.com/Azure/azure-powershell/blob/master/src/Network/Network.Test/ScenarioTests/ApplicationSecurityGroupTestsGenerated.ps1#L27
I think we'll add this function to Common.ps1 for the next release so it could be reused in every network test

Reply:
return variable is already of type Dictionary. Did you mean anything else?